core/src/main/java/org/phoenixctms/ctsms/domain/MaintenanceScheduleItemDaoImpl.java

```diff
@@ -115,16 +115,16 @@
 		Criteria maintenanceItemCriteria = createMaintenanceItemCriteria();
 		SubCriteriaMap criteriaMap = new SubCriteriaMap(MaintenanceScheduleItem.class, maintenanceItemCriteria);
 		if (inventoryId != null) {
-			maintenanceItemCriteria.add(Restrictions.eq("inventory.id", inventoryId.longValue()));
```

Comment:
jfyi by convention i always handover unboxed values to `Restriction.xx("property",x)`